public/app/plugins/datasource/elasticsearch/components/QueryEditor/index.tsx

```diff
@@ -50,47 +50,56 @@ const getStyles = (theme: GrafanaTheme2) => ({
 });
 
 interface Props {
-  value: ElasticsearchQuery;
+  value?: ElasticsearchQuery;
```

Comment:
forgot about this, i think this can safely be 
```suggestion
  value: ElasticsearchQuery;
```

and in https://github.com/grafana/grafana/pull/49529/files#diff-c1ad5e3467fa6c493747b6d65a0a86cab2cd02e64208358598062e6e2c810b72R99-R102 you can remove the optional chaining